backend/src/Designer/Controllers/AppDevelopmentController.cs

```diff
@@ -517,5 +518,13 @@ public ActionResult GetOptionListIds(string org, string app)
                 return NoContent();
             }
         }
+
+        [HttpGet("app-lib-version")]
+        public VersionResponse GetVersionOfTheAppLib(string org, string app)
+        {
+            string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
+            var version = _appDevelopmentService.GetAppLibVersion(AltinnRepoEditingContext.FromOrgRepoDeveloper(org, app, developer));
```

Comment:
Looks like appDevelopmentService.GetAppLibVersion can throw a NotFoundExecption. Would it cause this endpoint to return a 500 Server Error since we do not have to try to catch it here? 🤔

Reply:
Exception is handled in the filter. You can check GetVersionOfTheAppLibTests class. Every case is added in the test

Reply:
Thanks for clarification! 😊